src/test/groovy/org/sonarqube/gradle/SonarQubePluginTest.groovy

```diff
@@ -936,6 +943,18 @@ class SonarQubePluginTest extends Specification {
     return Paths.get(pathString).normalize().toAbsolutePath();
   }
 
+  private String pathList(Project project, String propertyName) {
```

Comment:
We should push this utility method further down in the file, away from the tests and closer to the other utility methods.